INSTALLING.md

```diff
@@ -15,8 +15,8 @@ Note the current compatability matrix:
 | Device              | OS              | Python   | Driver (TT-KMD)    | Firmware (TT-Flash)                        | TT-SMI                | TT-Topology                    |
 |---------------------|-----------------|----------|--------------------|--------------------------------------------|-----------------------|--------------------------------|
 | Grayskull           | Ubuntu 20.04    | 3.8.10   | v1.26              | fw_pack-80.4.0.0_acec1267.tar.gz (v4.0.0)  | v2.1.0 or above       | N/A                            |
```

Comment:
@tt-rkim as we're here, can we update Grayskull Firmware to [fw_pack-80.8.0.0.fwbundle](https://github.com/tenstorrent/tt-firmware/blob/main/fw_pack-80.8.0.0.fwbundle)?

Reply:
I believe v1.27.1 should be the right version for Grayskull too (that's referenced in the main TT-KMD repo) so we should update that too